test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -475,6 +482,12 @@ func (bp *BindPlugin) Bind(ctx context.Context, state *framework.CycleState, p *
 
 	bp.numBindCalled++
 	if bp.pluginInvokeEventChan != nil {
+		defer func() {
```

Comment:
These recover is necessary because we close the channel after all the test is done and we could hit `send on closed channel` panic.